src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: function(num) {
+      this.result += num;
+
+      return this;
+    },
+
+    subtract: function(num) {
+      this.result -= num;
+
+      return this;
+    },
+
+    multiply: function(num) {
+      this.result *= num;
+
+      return this;
+    },
+
+    divide: function(num) {
+      this.result /= num;
```

Comment:
number can be 0, so please add a additional condition for this case
return some error message when num is 0